2019-12-16 14:18:25 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: ERROR: relation "users" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:309)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:295)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:272)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:225)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:36)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:18:33 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: ERROR: relation "users" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:309)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:295)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:272)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:225)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:36)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:20:15 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: ERROR: relation "users" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:309)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:295)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:272)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:225)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:36)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:23:14 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: ERROR: relation "project0.users" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:309)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:295)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:272)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:225)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:36)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:24:14 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name emp_id was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:43)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:26:28 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:27:49 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 14:59:43 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 15:07:25 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:71)
2019-12-16 18:48:47 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:72)
2019-12-16 19:39:18 WARN  EmployeeDAOImpl:61 - Unable to retrieve all users
org.postgresql.util.PSQLException: The column name supervisor was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getInt(PgResultSet.java:2498)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:53)
	at com.revature.Driver.main(Driver.java:72)
2019-12-16 20:38:11 WARN  EmployeeDAOImpl:63 - Unable to retrieve all users 'employees' 
org.postgresql.util.PSQLException: The column name emp_type was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:2480)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:47)
	at com.revature.Driver.main(Driver.java:18)
2019-12-16 20:38:43 WARN  EmployeeDAOImpl:63 - Unable to retrieve all users 'employees' 
org.postgresql.util.PSQLException: The column name emp_type was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:2480)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:47)
	at com.revature.Driver.main(Driver.java:18)
2019-12-16 20:58:50 WARN  EmployeeDAOImpl:63 - Unable to retrieve all users 'employees' 
org.postgresql.util.PSQLException: The column name emp_type was not found in this ResultSet.
	at org.postgresql.jdbc.PgResultSet.findColumn(PgResultSet.java:2607)
	at org.postgresql.jdbc.PgResultSet.getString(PgResultSet.java:2480)
	at com.revature.repositories.EmployeeDAOImpl.findAll(EmployeeDAOImpl.java:47)
	at com.revature.Driver.main(Driver.java:18)
2019-12-17 20:45:52 WARN  EmployeesAdminDAOImpl:41 - Unable to verify if employee exists
org.postgresql.util.PSQLException: ERROR: relation "project0.employeesadmins" does not exist
  Position: 15
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:108)
	at com.revature.repositories.EmployeesAdminDAOImpl.employeeExists(EmployeesAdminDAOImpl.java:29)
	at com.revature.services.EmployeeAdminLogic.existingEmployeeCheck(EmployeeAdminLogic.java:199)
	at com.revature.services.EmployeeAdminLogic.existingUserLoginEmployee(EmployeeAdminLogic.java:38)
	at com.revature.services.Bank.startHere(Bank.java:64)
	at com.revature.Driver.main(Driver.java:10)
2019-12-17 22:43:50 WARN  CustomersDAOImpl:123 - 
org.postgresql.util.PSQLException: No value specified for parameter 5.
	at org.postgresql.core.v3.SimpleParameterList.checkAllParametersSet(SimpleParameterList.java:264)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:294)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at com.revature.repositories.CustomersDAOImpl.createCustomer(CustomersDAOImpl.java:120)
	at com.revature.services.CustomerLogic.setNewUserFields(CustomerLogic.java:81)
	at com.revature.services.Bank.startHere(Bank.java:83)
	at com.revature.Driver.main(Driver.java:10)
2019-12-17 23:26:35 WARN  CustomersDAOImpl:123 - ///
org.postgresql.util.PSQLException: No value specified for parameter 5.
	at org.postgresql.core.v3.SimpleParameterList.checkAllParametersSet(SimpleParameterList.java:264)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:294)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at com.revature.repositories.CustomersDAOImpl.createCustomer(CustomersDAOImpl.java:120)
	at com.revature.services.CustomerLogic.setNewUserFields(CustomerLogic.java:81)
	at com.revature.services.Bank.startHere(Bank.java:90)
	at com.revature.Driver.main(Driver.java:13)
